tests/test_sdba/test_adjustment.py

```diff
@@ -724,6 +726,226 @@ def test_real_data(self, open_dataset):
         new_scen.load()
 
 
+class TestOTC:
```

Comment:
Maybe add some `dask` testing (to chunk or not). See `TestQDM::test_mon_U` for example